### ISSUE TITLE
Fuzzing crash 654e6ed06597b86f23483b793664777da9b24c04

### ISSUE THREAD

**Issue by zimbatm** [nix-community/nixpkgs-fmt#98]
**Describe the bug**
Fuzzing crash: fuzz/artifacts/fmt/crash-654e6ed06597b86f23483b793664777da9b24c04

**To Reproduce**

1. Run `echo ".." > foo.nix`
2. Run `nixpkgs-fmt foo.nix'
```
thread 'main' panicked at 'assertion failed: prev.is_none()', src/engine/fmt_model.rs:269:9
```

**Comment by zimbatm** [comment]
@matklad what is the best way to submit fuzzing issues? I assume that we want to collect all those crashes in the repo?

**Comment by zimbatm** [comment]
I could create a new type of issue template for those if you want.

**Comment by matklad** [comment]
The best way is to send a PR which adds a `crash-` file. This issue in particular is closed by #100
